__tests__/pages/provinces/[provinceSlug]/index.test.tsx

```diff
@@ -28,6 +30,58 @@ describe("ProvincePage", () => {
     expect(breadcrumbs).toHaveAttribute("href", `/provinces/${province.slug}`);
     expect(title).toBeVisible();
   });
+
+  it("renders keyword input & filter value selected based on url parameter", () => {
+    const wrapper = render(
```

Comment:
```suggestion
    render(
```

You don't need the `wrapper` here, Mbak @ekamuktia.
Please remove it and use the `screen.getBy*` below.

Reply:
Okay, I'll update it. 
Please check the other test cases I added as well since I'm not familiar with jest and just added it mainly to pass the checks 😅